### ISSUE TITLE
the output seems something wrong？

### ISSUE THREAD

**Issue by dyh-bupt** [lucidrains/stylegan2-pytorch#74]
Hi, I'm using the stylegan2-pytorch to train a generator, with the dataset including 136k 256*256 images. (get from [网红人脸数据集](http://www.seeprettyface.com/mydataset_page3.html#mulu)

But, after 19 epoches, the results still seem something wrong：
![image](https://user-images.githubusercontent.com/50792949/83830636-3052ae00-a718-11ea-822d-7dc491576191.png)

What's the reason? Maybe the dataset is too large? 

Note: I'm using torch 1.15，and the command is "stylegan2_pytorch --data /data0/dyh/xinggan_face --name wanghong --results_dir /data0/dyh/CheckPoints/StyleGAN2/results --models_dir /data0/dyh/CheckPoints/StyleGAN2/models --image-size 256 --batch-size 2 --gradient-accumulate-every 8"

**Comment by lucidrains** [comment]
@dyh-bupt hmm, i'm not sure, I am running a 256x256 job right now and it seems fine. when you go through your results folder, do they all look like that? what are your losses?

**Comment by dyh-bupt** [comment]
thanks for your reply. All images in the results folder are like that. I'll try again. Is there any problem in my training command?

**Comment by lucidrains** [comment]
it looks correct. your pytorch looks a bit dated, so I would try updating to 1.5 before trying again

**Comment by dyh-bupt** [comment]
Oh, my torch is version 1.5, not 1.15, it's a slip in typing

**Comment by lucidrains** [comment]
i'm not sure then :( are you on the most recent version of stylegan2_pytorch?

**Comment by dyh-bupt** [comment]
yes, I install the stylegan2_pytorch just 4 days ago. I'm training on two new dataset. I'll see the results first and I'll tell you wether they are good or bad. Thanks for your timely reply~

**Comment by Johnson-yue** [comment]
@lucidrains @dyh-bupt  I am using pytorch==1.5, too.
what is your pytorch version ? @lucidrains 

we use pytorch==1.5 version because the package "kornia==0.3.1" requirement pytorch==1.5.0

**Comment by dyh-bupt** [comment]
@Johnson-yue @lucidrains 
Yes, the default torch version is 1.5 with the command "pip install stylegan2_pytorch".
After some experiments, the model sometimes works but sometimes break down.
Especially, on one dataset, when I use the complete dataset(with 136k images), the results are as follows:
![image](https://user-images.githubusercontent.com/50792949/84220793-47bddc80-ab06-11ea-976e-9c60d184bfe4.png)
But when I use a new dataset by randomly selecting 13k images from that complete dataset, the results seems acceptable. 
Oh, that's crazy!

**Comment by Johnson-yue** [comment]
less dataset can get better result？？ what is the problem？？

**Comment by dyh-bupt** [comment]
> less dataset can get better result？？ what is the problem？？

I don't know what happend. Maybe different datasets need different hyper-parameters when training by command?
What's your training results? Do you have the same results with me?

**Comment by lucidrains** [comment]
@dyh-bupt great results! i'll have to reexamine how the images are loaded, perhaps it's a problem with the dataset class

**Comment by lucidrains** [comment]
@dyh-bupt are you ok with me adding that result of yours into the samples folder of this repository? just so everyone can see in the readme?

**Comment by dyh-bupt** [comment]
@lucidrains Thanks for you reply.
Of course, you can add my results into the samples folder. It's my pleasure~
If you can find what's the problem in the code, I hope you would tell me, thank you~
At last, StyleGAN (and version2) is great and funny ! ^.^

**Comment by lucidrains** [comment]
I'll look into it!

**Comment by Johnson-yue** [comment]
@dyh-bupt  I used the flower102 dataset , this is author used , and the result in ReadMe, but I can not get the same result with default configuration

**Comment by dyh-bupt** [comment]
> @dyh-bupt I used the flower102 dataset , this is author used , and the result in ReadMe, but I can not get the same result with default configuration

I cannot understand your question clearly. Maybe you can @ the author and describe your problem in more detail, with the dataset info, the command you use and the env you use (e.g. the torch version).

**Comment by Johnson-yue** [comment]
@dyh-bupt  the author used the flower102 dataset and got a good result like ：
![image](https://user-images.githubusercontent.com/10268274/84612706-05284580-aef4-11ea-9752-97a25c76bc43.png)

But I used the same dataset and default configuration and hyper-parameters but got not good result， this is my question。

**Comment by dyh-bupt** [comment]
> @dyh-bupt the author used the flower102 dataset and got a good result like ：
> ![image](https://user-images.githubusercontent.com/10268274/84612706-05284580-aef4-11ea-9752-97a25c76bc43.png)
> 
> But I used the same dataset and default configuration and hyper-parameters but got not good result， this is my question。

I didn't do this experiment. Maybe you should @ the author. ^.^
